### ISSUE TITLE
401 -Unauthorized error for jhoeller/password for rest application

### ISSUE THREAD

**Issue by vsk106** [spring-guides/tut-rest#24]
I have started Spring boot rest application and trying to hit from postman with username/password as jhoeller/password under Basic Aauth.It is giving error as 401 -Unauthorized. 

Could you please help me to resolve this.

**Comment by issmith1** [comment]
I duplicate this but its on Spring 3 version.
 I moved to other tutorials and gs that are on Spring 4.1

**Comment by visaxin** [comment]
ME TOO.
Just fail login in Chrome http://localhost:8080

**Comment by issmith1** [comment]
use the Spring 4 tutorials.  They have a IMO dispersed hard to follow organization, however, they are good.   Start at root https://github.com/spring-projects/spring-security-oauth.
Descend to root/samples,  or root/tests/annotations.  Look for the READMEs in various places.  

We got our oauth2 RS and AS running and are developing good use cases for the oauth2 flows. 
hope this helps.

**Comment by gregturn** [comment]
This tutorial was completely rewritten as of https://github.com/spring-guides/tut-rest/commit/63d393748ec27b072854f8b35a763e27ac6e0983.

I suggest you read it from the top and be sure to let us know if you spot any issues!
